README.md

```diff
@@ -61,8 +61,6 @@
    " Keep Plugin commands between vundle#begin/end.
    " plugin on GitHub repo
    Plugin 'tpope/vim-fugitive'
-   " plugin from http://vim-scripts.org/vim/scripts.html
```

Comment:
Instead of removing I think a better way would be to comment out the next line and the `ascenator/L9` line.